test/categorical_algebra/DataMigrations.jl

```diff
@@ -318,12 +318,26 @@ X = @acset UndirectedBipartiteGraph begin
   tgt = [1,1,2,3]
 end
 
-Y = ΣF(X)
+Y,α = ΣF(X)
 @test nparts(Y, :V) == 7
 @test nparts(Y, :E) == 4
 @test length(Y[:src] ∩ Y[:tgt]) == 0
+@test isempty(collect(α[:V₁]) ∩ collect(α[:V₂]))
 
-@test SigmaMigration(idF, Graph, Graph)(Y) == Y
+@test first(SigmaMigration(idF, Graph, Graph)(Y)) == Y
+
+
+idF = FinFunctor(
+  Dict(VG => VG, EG => EG, :Weight=>:Weight),
+  Dict(srcG => srcG, tgtG => tgtG, :weight=>:weight),
+  SchWeightedGraph, SchWeightedGraph
+)
+Y = @acset WeightedGraph{Symbol} begin 
+  V=2; E=2; Weight=1; src=1; tgt=[1,2]; weight=[AttrVar(1), :X] 
+end
+@test_throws ErrorException SigmaMigration(idF, WeightedGraph{Symbol}, WeightedGraph{Symbol})
+ 
+# @test ΣF(Y) == Y # TODO allow homs between attrtypes
```

Comment:
Should this be an issue?

Reply:
Created one! https://github.com/AlgebraicJulia/Catlab.jl/issues/788